src/main/java/org/cloudbees/literate/jenkins/LiterateBranchProject.java

```diff
@@ -229,7 +229,11 @@ public synchronized SCM getScm() {
     @Override
     @NonNull
     public SCMCheckoutStrategy getScmCheckoutStrategy() {
-        return Jenkins.getInstance().getDescriptorByType(SCMCheckoutStrategyImpl.DescriptorImpl.class).getInstance();
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            throw new IllegalStateException(); // TODO 1.590+ getActiveInstance
```

Comment:
please add a message - if a user ever sees this having ("Jenkins is starting up or shutting down") is highly valuable.